neore/scripts/package.sh

```diff
@@ -67,6 +67,10 @@ function download_flume(){
     rm -f ${FLUME_WO_TAR}/lib/libthrift-*.jar
     mv ${LIBTHRIFT_JAR} ${FLUME_WO_TAR}/lib
 
+    _log "### Disable httpclient and httpcore libraries distributed within apache-flume bundle... ###"
+    mv ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar.old
```

Comment:
What is the relationship between this and the topic of the PR (Dynamoc sink)? Has this change any implact in installation from source documenation?